parl/env/continuous_wrappers.py

```diff
@@ -0,0 +1,45 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+import numpy as np
+
+
+class ActionMappingWrapper(gym.Wrapper):
+    def __init__(self, env):
+        """Mapping action space [-1, 1] of model output 
+        to new action space [low_bound, high_bound].
+        """
+        gym.Wrapper.__init__(self, env)
+        assert isinstance(self.env.action_space, gym.spaces.Box)
+        self.low_bound = self.env.action_space.low[0]
+        self.high_bound = self.env.action_space.high[0]
+        assert self.high_bound > self.low_bound
+
+    def reset(self, **kwargs):
+        return self.env.reset(**kwargs)
+
+    def step(self, model_output_act):
+        """
+        Args:
+            model_output_act: np.array, which value is in [-1, 1]
```

Comment:
model_output_act(np.array):  The values must be in in [-1, 1].

Reply:
done